build.js

```diff
@@ -178,12 +178,21 @@ function handle(options, compiler) {
   var skeleton = fs.readFileSync(__dirname + path.sep + options.skeleton_file, 'utf-8');
   var html = dataToHtml(skeleton, options.browsers, options.tests, options.compiler);
 
+  var createIterableObject = [];
+  try {
+    var compiled = options.compiler(
+      'global.__createIterableObject = function(a, b, c) { return function*() { yield a; yield b; yield c; }(); };'
```

Comment:
Isn't the point of `__createIterableObject` that it should work on platforms that support either `Symbol.iterator`, or generators, as both are valid means of creating iterables? This doesn't have the `Symbol.iterator` fallback.